tests/FSharp.Test.Utilities/ProjectGeneration.fs

```diff
@@ -32,6 +32,8 @@ open FSharp.Compiler.Text
 open Xunit
 open FSharp.Test.Utilities
 
+open FSharp.Test.Utilities
```

Comment:
```suggestion
```

In general, this all is getting into very good shape, I think we can start minimizing diff and attracting more reviews :)